dpnp/backend/extensions/blas/dot_common.hpp

```diff
@@ -23,30 +23,26 @@
 // THE POSSIBILITY OF SUCH DAMAGE.
 //*****************************************************************************
 
-#include <pybind11/pybind11.h>
+#pragma once
+
+#include <oneapi/mkl.hpp>
```

Comment:
Why do you remove `#include <pybind11/pybind11.h>`? We are still using `namespace py = pybind11;` below. Should not we keep the include?